### DISCUSSION TITLE
The deployment is successful but the chatbot shows up on apiapp link. Is this correct?

### DISCUSSION THREAD

**Discussion by kacheng24** [microsoft/chat-copilot#1093] [Q&A]
We have created our own **api**app and **web**app services. After the deployment, the chatbot shows up on **api** site. Seems like there is no file deployed into the webapp. The bot is working fine. It let me upload doc and answer our questions using the uploaded docs. We want to know if the front-end is supposed to be in the api? Did we make mistake in the setting? 
![image](https://github.com/user-attachments/assets/d6f33bf7-fefa-4681-90f3-c34ab9385761)

When we looked into the Azure, we see no file for the webapp 
![image](https://github.com/user-attachments/assets/fb348600-b40c-4c02-90f0-fae07d87fef4)

We do see files for apiapp 
![image](https://github.com/user-attachments/assets/cacf9984-9f03-4a00-aa52-62e3d31aba47)

This is a code in the main.bicep. Since we want to use the existing web/api services, we changed the name in the script. 
Should we put the **api**app name or **web**app name in here?
![image](https://github.com/user-attachments/assets/2ef5a772-0b5c-417e-925a-211332213482)

The Azure WebApp name is apiapp when the script was being run.
![image](https://github.com/user-attachments/assets/b307ac22-9cdc-4908-8e6e-2cf53dead332)

Please let us know if you have any ideas. Thank you!

**Comment by aaronba** [comment]
Hey @kacheng24 , When you call https://github.com/microsoft/chat-copilot/blob/main/.github/workflows/copilot-build-backend.yml from GitHub Actions, it calls 

https://github.com/microsoft/chat-copilot/blob/main/scripts/deploy/package-webapi.ps1

And specifically on these lines it builds the frontend and copies it into the .zip webapi deploy package. (zip)

`
if (-Not $SkipFrontendFiles) {
    Write-Host "Building static frontend files..."

    Push-Location -Path "$PSScriptRoot/../../webapp"
    
    $filePath = "./.env.production"
    if (Test-path $filePath -PathType leaf) {
        Remove-Item $filePath
    }
    
    Add-Content -Path $filePath -Value "REACT_APP_BACKEND_URI="
    Add-Content -Path $filePath -Value "REACT_APP_SK_VERSION=$Version"
    Add-Content -Path $filePath -Value "REACT_APP_SK_BUILD_INFO=$InformationalVersion"

    Write-Host "Installing yarn dependencies..."
    yarn install
    if ($LASTEXITCODE -ne 0) {
        exit $LASTEXITCODE
    }

    Write-Host "Building webapp..."
    yarn build
    if ($LASTEXITCODE -ne 0) {
        exit $LASTEXITCODE
    }

    Pop-Location

    Write-Host "Copying frontend files to package"
    Copy-Item -Path "$PSScriptRoot/../../webapp/build/*" -Destination "$publishOutputDirectory\wwwroot" -Recurse -Force
}`

**Reply by kacheng24** [reply]
Thank you for answering our question, but we did leave the SkipFrontendFiles by default so it should not have skipped it.
We see zip files in the out folder
![image](https://github.com/user-attachments/assets/cf217e21-a641-4d36-857e-4f052403eba1)
